src/scripts/main.js

```diff
@@ -1,3 +1,113 @@
 'use strict';
 
-// write code here
+const buttons = Array.from(document.querySelectorAll('button'));
+const tableBody = document.querySelector('.field tbody');
+const trElements = document.querySelector('tr');
```

Comment:
The variable `trElements` is assigned a single `tr` element instead of a collection. If you intend to use it for iterating over all `tr` elements, consider using `document.querySelectorAll('tr')`.